libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/coordinator/RoadNameLabelCoordinator.kt

```diff
@@ -14,6 +14,6 @@ internal class RoadNameLabelCoordinator(
 ) : UICoordinator<ViewGroup>(roadNameLabelLayout) {
 
     override fun MapboxNavigation.flowViewBinders(): Flow<UIBinder> {
-        return navigationViewContext.flowUiBinder { it.roadName }
+        return navigationViewContext.flowUiBinder({ it.roadName })
```

Comment:
```suggestion
        return navigationViewContext.flowUiBinder { it.roadName }
```

Reply:
We can't do that. `flowUiBinder` function accepts 2 lambdas.

```kotlin
internal fun <T : UIBinder> DropInNavigationViewContext.flowUiBinder(
    selector: (value: NavigationUIBinders) -> StateFlow<T>,
    mapper: suspend (value: T) -> T = { it }
): Flow<T>
```

Reply:
confused why we need it to accept 2 lambdas 🤔

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/5580/files#diff-0c396534cd7280dbe0dd7296f085184ff623d99b10e52ac460beedff2289b21f